clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/config/GoogleConfigurationProperties.groovy

```diff
@@ -30,11 +30,13 @@ class GoogleConfigurationProperties {
     List<String> imageProjects
     ConsulConfig consul
     String userDataFile
+    List<String> regions
   }
 
   List<ManagedAccount> accounts = []
   int asyncOperationTimeoutSecondsDefault = ASYNC_OPERATION_TIMEOUT_SECONDS_DEFAULT
   int asyncOperationMaxPollingIntervalSeconds = ASYNC_OPERATION_MAX_POLLING_INTERVAL_SECONDS
   List<String> baseImageProjects
   long maxMIGPageSize = 50
+  List<String> defaultRegions
```

Comment:
Added comments and updated behavior so that an empty list will index no regions. Didn't update the names because we decided this will have some affect on mutating calls as well.